src/unit_tests/wazuh_db/test_wdb_integrity.c

```diff
@@ -1526,6 +1527,156 @@ void test_wdbi_last_completion_step_fail(void **state)
 
 }
 
+// Test wdb_global_group_hash_cache
+
+void test_wdb_global_group_hash_cache_clear_success(void **state)
```

Comment:
We cold write the cache, before testing the clear command.

Reply:
Done